src/index.js

```diff
@@ -86,7 +86,10 @@ async function startApp() {
     }
   }
 
-  schema = applyGraphQLMiddleware(schema, graphqlTimeoutMiddleware(RESOLVER_TIMEOUT_MS))
+  if (RESOLVER_TIMEOUT_MS > 0) {
+    console.warn("[FEATURE] Enabling resolver timeouts")
+    schema = applyGraphQLMiddleware(schema, graphqlTimeoutMiddleware(RESOLVER_TIMEOUT_MS))
```

Comment:
If I understand this correctly - if setting `RESOLVER_TIMEOUT_MS=0`, `graphqlTimeoutMiddleware` will use a `timeoutForField` - is this set on the schema?

Reply:
If `RESOLVER_TIMEOUT_MS` is set to `0` then the timeout middleware doesn't get applied to the schema at all.

Reply:
Ach. Duh.